.github/workflows/test-deploy-linux.yml

```diff
@@ -0,0 +1,23 @@
+name: Test set-up on Linux
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set-up Docker
+      run: |
+        make install-docker-ubuntu
+    - name: Initialize environment
+      run: |
+        make init-env
```

Comment:
Right now this failure seems to be because of the same issue described in #137 
It might be that the DB container does not initialize in time (30 sec.). We could change the timeout to be 60 on line 29 of `./api/wait_for_db.py`

Reply:
Seems that it didn't solve it